eventual-cassandra/src/test/scala/com/evolutiongaming/kafka/journal/eventual/cassandra/ReplicatedCassandraTest.scala

```diff
@@ -38,7 +38,7 @@ class ReplicatedCassandraTest extends AnyFunSuite with Matchers {
   private val origin          = Origin("origin")
   private val version         = Version.current
   private val uuid            = UUID.randomUUID
-  private val recordId        = RecordId.unsafe(uuid.toString)
+  private val recordId        = RecordId(uuid)
```

Comment:
here we use default `apply` in `unsafe` fashion, right?

Reply:
no, default `apply` is not unsafe because just creating new instance of case class **is not unsafe**. What is actually unsafe is `UUID.randomUUID`: its part of `RecordId.of: F[RecordId]` and `RecordId.unsafe: RecordId` where unsafe action hidden under effect or factory method explicitly tells you that its unsafe.